gtk4-macros/Cargo.toml

```diff
@@ -14,9 +14,12 @@ rust-version = "1.57"
 [lib]
 proc-macro = true
 
+[features]
+xml_validation = ["quick-xml"]
```

Comment:
Shouldn't this be the default?

Reply:
It is only useful for xml files defined inside the source file (using `#[template (string = #'....')]`) which no ones uses in a real app. Most of people either uses a file or a gresource